src/blocks/net.rs

```diff
@@ -65,17 +65,17 @@ pub async fn run(
     let mut tx_hist = [0f64; 8];
     let mut rx_hist = [0f64; 8];
 
+    // Get interface name
+    let interface = block_config
+        .interface
+        .clone()
+        .or_else(default_interface)
+        .unwrap_or_else(|| "lo".to_string());
+
     loop {
         let mut speed_down: f64 = 0.0;
         let mut speed_up: f64 = 0.0;
 
-        // Get interface name
-        let interface = block_config
-            .interface
-            .clone()
-            .or_else(default_interface)
-            .unwrap_or_else(|| "lo".to_string());
```

Comment:
The interface can change over time, so it has to be inside loop

Reply:
Oh crap sorry, I didn't intend to push this change, I was just trying to fix some issue with my local config, for some reason `default_interface` didn't return a correct value.

`default_interface` is probably doing some blocking I/O though? Should be at least mark it with a TODO?

Reply:
> default_interface is probably doing some blocking I/O though? Should be at least mark it with a TODO?

Yeah, [`neli`](https://github.com/jbaublitz/neli) supports `async`/`tokio`, but I didn't have enough time to figure out how to use it

Reply:
> or some reason default_interface didn't return a correct value

That's interesting. The returned interface name didn't match the output of `ip route show default`?

Reply:
Can you open a new issue regarding incorrect `default_interface`?